### DISCUSSION TITLE
Last changes to Home Assistant Script?

### DISCUSSION THREAD

**Discussion by carsten-h** [tteck/Proxmox#3965] [General] [answered]
Hello!

One of the last changes for the Home Assistant script is this:
"Home Assistant has transitioned to using uv for managing the virtual environment and installing additional modules."

Do we have to change something in older running VM with Home Assistant for this?

**Comment by tteck** [comment, marked as answer]
Only affects Home Assistant Core. 

https://github.com/tteck/Proxmox/discussions/3836
